drivers/ovsd/ovsSwitch.go

```diff
@@ -359,7 +364,12 @@ func (sw *OvsSwitch) CreatePort(intfName string, cfgEp *mastercfg.CfgEndpointSta
 
 	// Set the link mtu to 1450 to allow for 50 bytes vxlan encap
 	// (inner eth header(14) + outer IP(20) outer UDP(8) + vxlan header(8))
-	err = setLinkMtu(intfName, vxlanEndpointMtu)
+	if sw.netType == "vxlan" {
```

Comment:
Is there a constant you can use here instead of a string literal?

Reply:
There is no constant available ; same string literal has been used other part of code

Reply:
see this section of code:
 sw.ovsdbDriver.ovsSwitch = sw

        if netType == "vxlan" {
                ofnetPort = vxlanOfnetPort
                ctrlrPort = vxlanCtrlerPort
                switch fwdMode {
                case "bridge":
                        datapath = "vxlan"
                case "routing":
                        datapath = "vrouter"
                default: